dvc/command/repro.py

```diff
@@ -57,7 +57,8 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    REPRO_HELP = "Check for changes and reproduce stages and dependencies."
+    REPRO_HELP = "Reproduce complete or partial pipelines"
+    " by executing their stages."
```

Comment:
maybe `running`?

Reply:
Since we have ` dvc run` command, we use the term "execute" a lot in the repro cmd ref (https://dvc.org/doc/command-reference/repro).